src/se/terhol/pisemka32/VendorImpl.java

```diff
@@ -0,0 +1,79 @@
+package se.terhol.pisemka32;
+
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.util.Collection;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
+/**
+ * @author Tereza Holm
+ */
+public class VendorImpl implements Vendor {
+    private SortedMap<Magazine, Double> myMagazines = new TreeMap<>();
```

Comment:
What is a reason for name `myMagazines`? If there is any, I would rather name it just `magazine` to not confuse a reader.

Reply:
Done